tests/review-child-handoff.test.sh

```
@@ -60,6 +112,47 @@ simplify_context_out="$(uberdev_agent_context_create "$TMP/simplify-run" "$simpl
+import json,sys
+path,manifest_path=sys.argv[1:]
+limit=json.load(open(manifest_path))['input_limits']['max_serialized_bytes']
+with open(path,encoding='utf-8') as stream:
+    value=json.load(stream)
+value['inputs']['changed_paths'][-1]=value['inputs']['changed_paths'][-1][:-3]+'x.ts'
+serialized=json.dumps(value['inputs'],sort_keys=True,separators=(',',':')).encode()
+assert len(serialized)==limit+1
+with open(path,'w',encoding='utf-8') as stream:
+    json.dump(value,stream,separators=(',',':'))
+PY
+! _uberdev_child_prepare review_pr.review.correctness "$MAX_PLUS_ONE_HANDOFF" "$UBERDEV_CHILD_RESULT" "$UBERDEV_CHILD_STATUS" dispatch >/dev/null 2>&1
```

Shellcheck (0.11.0)

[info] This ! is not on a condition and skips errexit. Use `&& exit 1` instead, or make sure $? is checked.

(SC2251)